playground/webpack.shared.config.js

```diff
@@ -23,6 +23,10 @@ const MONACO_URL = {
   dev: 'https://cdnjs.cloudflare.com/ajax/libs/monaco-editor/0.14.3/min/',
   production: 'https://cdnjs.cloudflare.com/ajax/libs/monaco-editor/0.14.3/min/'
 };
+const REACT_MEDIA_URL = {
```

Comment:
You need to list `react-media` as an external (line 89) and include include these URLs in the HTML template (line 109).

Reply:
`react-media` isnt being used since we switched to simple CSS media queries. Should I delete `REACT_MEDIA_URL`?

Reply:
If it's not actually being used, then yeah